geocity/apps/permits/templatetags/permits_extras.py

```diff
@@ -40,6 +40,9 @@ def permit_request_summary(context, permit_request):
     documents = services.get_permit_complementary_documents(
         permit_request, user=context.request.user
     )
+    is_validator = services.has_permission_to_validate_permit_request(
```

Comment:
Est-ce que le validateur peut voir le champs même si le permitrequest n'est pas en validation. Au besoin utiliser également `can_validate_permit_request`

Reply:
Exactly, with `can_validate_permit_request` it was only true if the request was in validation, it's why I used `has_permission_to_validate_permit_request` in order not to depend on the status.